server/channels/api4/user.go

```diff
@@ -2262,6 +2263,38 @@ func attachDeviceId(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
+func setIgnoreACK(c *Context, w http.ResponseWriter, r *http.Request) {
+	props := model.MapFromJSON(r.Body)
+	ignore := props["ignore"]
+	if ignore != "false" && ignore != "true" {
+		c.SetInvalidParam("ignore")
+		return
+	}
+
+	currentSession := c.AppContext.Session()
+	if currentSession.DeviceId == "" {
+		ReturnStatusOK(w)
+		return
+	}
+
+	if currentSession.Props[model.SessionPropIgnoreNotificationACK] == ignore {
+		ReturnStatusOK(w)
+		return
+	}
+
+	value := false
+	if ignore == "true" {
+		value = true
+	}
```

Comment:
nit: Seems that the first thing `SetIgnoreNotificationACK` does it converts the boolean back to string value. Given that we checked above if it's a valid value, do we need to convert to bool here?